targets/targets.json

```diff
@@ -6278,7 +6287,20 @@
         "overrides": {
             "deep-sleep-latency": 1,
             "tickless-from-us-ticker": true
-        }
+        },
+        "supported_c_libs": {
+            "arm": [
+                "std",
+                "small"
+            ],
+            "gcc_arm": [
+                "std",
+                "small"
+            ]
+        },
+        "supported_application_profiles": [
+            "bare-metal"
```

Comment:
Can you not build with the full profile?

Reply:
You should be able to, I'm working on changing this and will confirm once I have tested a full build. 👍🏿